contentful/resource_builder.py

```diff
@@ -29,12 +29,14 @@ def __init__(
             localized,
             json,
             includes_for_single=None,
-            depth=0):
+            depth=0,
```

Comment:
This is not ma method call, but a method definition, therefore default should be 0.